doc/site/guides/lua-language-server.markdown

```diff
@@ -0,0 +1,243 @@
+---
+layout: default
+title: Lua Language Server
+parent: Guides
+permalink: guides/lua-language-server
```

Comment:
I mean, if you want, add yourself as `author: yournick`.

Then add a file in `_data/authors` as the others

Reply:
55ee478c0cf4e253ed6ca49a0e2c70bf0fc78195